OpenRA.Mods.Common/Widgets/Logic/Installation/InstallFromCDLogic.cs

```diff
@@ -51,12 +52,26 @@ bool IsValidDisk(string diskRoot)
 			return installData.DiskTestFiles.All(f => File.Exists(Path.Combine(diskRoot, f)));
 		}
 
+		bool IsTFD(string diskpath)
+		{
+			bool test = File.Exists(Path.Combine(diskpath, "data1.hdr"));
+			int i = 0;
+			while (test && i < 14)
```

Comment:
You can simplify this down to `return TFDArchives.All(x => File.Exists(Path.Combine(diskpath, x)));` (using my earlier rename request).